src/oci.rs

```diff
@@ -150,19 +156,32 @@ fn append_dir_all_with_xattrs(
             continue;
         }
 
-        if entry.file_type().is_dir() || entry.file_type().is_symlink() {
+        if entry.file_type().is_symlink() {
+            if meta.mtime() > clamp_mtime {
```

Comment:
Should rpmoci support pinning to future times?

Reply:
My interprtation of this question is:

by setting source_date_epoch on the dnf install we are telling dnf/rpm to use that date for files installed via package manager. post install scripts can still create files/symlinks etc with  other timestamps - if a time in the future is set, then the build won't be reproducible since we won't change the time. if we changed all file mtimes to source date epoch then we'd be able to support pinning to future times. should we support that?

I don't care about that scenario at present, but thanks for flagging.